src/System.Memory/tests/Performance/Perf.Span.IndexOf.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+namespace System.Memory.Tests
+{
+    public class Perf_Span_IndexOf
+    {
+        private const int InnerCount = 100000;
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        [InlineData(1)]
+        [InlineData(10)]
+        [InlineData(100)]
+        [InlineData(1000)]
+        [InlineData(10000)]
```

Comment:
Removing some, like 10k is fine. However, it is important to have the 1, 10, 100, to see the overhead cost of vectorization, imo. Will remove the 10k.